### PR TITLE
fix: new landscape link ref warning, edit landscape link not working,  typo, vestigial code

### PR DESCRIPTION
**Pull request by mmnoo**

### CODE REVIEW

File: mrtt-ui/src/components/EditLink.js [hunk 1]
```diff
@@ -2,13 +2,13 @@ import { Edit } from '@mui/icons-material'
 import React from 'react'
 
 import language from '../language'
-import { ButtonSecondary } from '../styles/buttons'
+import { LinkLooksLikeButtonSecondary } from '../styles/typography'
 
 const EditLink = (props) => {
   return (
-    <ButtonSecondary {...props}>
+    <LinkLooksLikeButtonSecondary {...props}>
```
Comment (mmnoo) [inline mrtt-ui/src/components/EditLink.js:9]: ButtonSecondary isnt  a link to the `to` prop doesnt work on it.

File: mrtt-ui/src/views/Landscapes.js [hunk 2]
```diff
@@ -90,9 +96,9 @@ function Landscapes() {
     <ContentWrapper>
       <TitleAndActionContainer>
         <PageTitle>{language.pages.landscapes.title}</PageTitle>
```
Comment (mmnoo) [inline mrtt-ui/src/views/Landscapes.js:98]: This one worked, but with a warning about how component isnt a legal prop and something about refs

Comment (julia-rom) [reply mrtt-ui/src/views/Landscapes.js:98]: so is there still a warning displaying? was it there before too? @mmnoo

Comment (mmnoo) [reply mrtt-ui/src/views/Landscapes.js:98]: no warning anymore. sorry that was worded unclearly. This removed the warning.

**Review by julia-rom — APPROVED** [review]

**Pull request merged**